PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/platformPlugin.xml

```diff
@@ -16,6 +16,7 @@
         <li>.asm endpoint generation for non-default locale - <a href="https://github.com/JetBrains/azure-tools-for-intellij/issues/212">#212</a></li>
         <li>Publish Azure web app does not work - <a href="https://github.com/JetBrains/azure-tools-for-intellij/issues/236">#236</a></li>
         <li>Fix exception on SQL Server create - <a href="https://github.com/JetBrains/azure-tools-for-intellij/issues/224">#224</a></li>
+        <li>Debugging an entire function app is no longer possible - <a href="https://github.com/JetBrains/azure-tools-for-intellij/issues/238">#238</a></li>
```

Comment:
I would replace the whole <change-note> section with bug fix notes only since it is not the replacement, but just a bug fix to not confuse with release notes.

Reply:
Fixed.